src/api/procedures/moveFunds.ts

```diff
@@ -0,0 +1,138 @@
+import { PalletConfidentialAssetConfidentialMoveFunds } from '@polkadot/types/lookup';
+import { ISubmittableResult } from '@polkadot/types/types';
+import { Identity } from '@polymeshassociation/polymesh-sdk/api/entities/Identity';
+import { ErrorCode } from '@polymeshassociation/polymesh-sdk/types';
+import { TransactionSpec } from '@polymeshassociation/polymesh-sdk/types/internal';
+import { identityIdToString } from '@polymeshassociation/polymesh-sdk/utils/conversion';
+import { filterEventRecords } from '@polymeshassociation/polymesh-sdk/utils/internal';
+
+import { ConfidentialProcedure } from '~/base/ConfidentialProcedure';
+import { ConfidentialAccount, Context, PolymeshError } from '~/internal';
+import { ConfidentialLegProof, ConfidentialProcedureAuthorization, TxTags } from '~/types';
+import { ExtrinsicParams } from '~/types/internal';
+import {
+  meshProofsToConfidentialLegProof,
+  meshPublicKeyToKey,
+  serializeAssetMoves,
+} from '~/utils/conversion';
+
+export type MoveFundsParams = {
+  from: ConfidentialAccount;
+  to: ConfidentialAccount;
+  proofs: ConfidentialLegProof[];
+};
```

Comment:
```suggestion
export type MoveFundsParams = {
  from: string | ConfidentialAccount;
  to: string | ConfidentialAccount;
  proofs: ConfidentialLegProof[];
};
```
Also, since this type is used while defining the method in `ConfidentialAccounts`, this needs to be exported from `~/api/procedure/types`